src/db/timezones.ts

```diff
@@ -0,0 +1,44 @@
+import { userModel } from "../models/user"
+import { time } from "discord.js"
+async function addTimeZone(timezone: String, userID: String) {
+  // Find the user
+  const userObject: any = await userModel.findOne({ id: userID })
+
+  if (!userObject) {
+    return "you are not verified"
+  }
+
+  // Update the user
+  userObject.timezone = timezone
+
+  // Save the user
+  await userObject.save()
+
+  return `Your timezone has been set to ${timezone}`
+}
+
+async function compareTimezones(firstUserId: string, secondUserId: string) {
+  const [firstUser, secondUser] = await Promise.all([
+    userModel.findOne({ id: firstUserId }),
+    userModel.findOne({ id: secondUserId }),
+  ])
+
+  if (!firstUser || !secondUser) {
+    return "One or both of the users is not verified"
+  }
+
+  if (!firstUser.timezone || !secondUser.timezone) {
+    return "One or both of the users don't have a timezone set"
+  }
+
+  const date = "Sun, 30 May 2021 14:59:15"
```

Comment:
What purpose of this variable?

Reply:
that was done by @mmvergara

I think it is needed so both users are at the same time just different time zones so it can be compared.

Reply:
its used 2 time so he might have just thought to keep it in a variable

Reply:
This also can be replaced with ``
```suggestion
  const date = new Date().toUTCString()
```

Reply:
I will add my bot to the discord so I can test everything since I need 2 people to test compare

Reply:
Done